cdk/lib/__snapshots__/registration.test.ts.snap

```diff
@@ -193,7 +193,7 @@ Object {
             "ReturnData": false,
           },
         ],
-        "Threshold": 0.014400000000000013,
+        "Threshold": 0.07200000000000006,
```

Comment:
These thresholds are [automatically calculated](https://github.com/guardian/cdk/blob/9d5ff3552e1b95391415c4c0db0a0aac90d34f90/src/experimental/constructs/error-budget-alarm.ts#L173) by multiplying the error budget by various burn rates ([slow](https://github.com/guardian/cdk/blob/9d5ff3552e1b95391415c4c0db0a0aac90d34f90/src/experimental/constructs/error-budget-alarm.ts#L153)/[medium](https://github.com/guardian/cdk/blob/9d5ff3552e1b95391415c4c0db0a0aac90d34f90/src/experimental/constructs/error-budget-alarm.ts#L136)/[fast](https://github.com/guardian/cdk/blob/9d5ff3552e1b95391415c4c0db0a0aac90d34f90/src/experimental/constructs/error-budget-alarm.ts#L120)).